app/views/ExposureHistory/DetailedHistory.js

```diff
@@ -14,60 +19,94 @@ import { SingleExposureDetail } from './SingleExposureDetail';
  */
 export const DetailedHistory = ({ history }) => {
   const { detailedHistoryPageWhatThisMeansPara } = useAssets();
+  const { hasBeenExposed } = useContext(ExposureNotificationContext);
   const exposedDays = history.filter((day) => day.exposureMinutes > 0);
 
-  return (
-    <>
-      <ExposureCalendarView weeks={3} history={history} />
+  const Divider = () => {
+    return <View style={styles.divider} />;
+  };
 
-      <Divider />
+  const gpsExposureInfo = () => {
```

Comment:
the existing view layout is tightly couple to gps logic, instead of untangling it all, i moved it over here for sake of time. a future pass should consolidate this. none of the functionality was changed, so for gps this is the same as it was before.